core/tests/karma.conf.ts

```diff
@@ -91,7 +92,7 @@ module.exports = function(config) {
         html: { outdir: 'html' }
       }
     },
-    autoWatch: true,
+    autoWatch: isDocker ? false : true,
```

Comment:
Add a comment above this line explaining the rationale.

Reply:
Reverted.